charts/kserve-resources/templates/localmodelnode/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.kserve.localmodel.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kserve-localmodelnode-controller-manager
+  namespace:  {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: kserve-localmodelnode-controller-manager
+    control-plane: kserve-localmodelnode-controller-manager
+    controller-tools.k8s.io: "1.0"
+spec:
+  selector:
+    matchLabels:
+      control-plane: kserve-localmodelnode-controller-manager
+      controller-tools.k8s.io: "1.0"
+  template:
```

Comment:
I think we want to only run the daemonset one on some nodes. e.g. certain GPU types. Then we need to make `matchLabels` configurable in the helm chart. But we can do it in a different PR.

https://kubernetes.io/docs/tasks/manage-daemon/pods-some-nodes/#step-2-create-the-manifest

Reply:
Can we create an issue to track so we do not forget